frame.go

```diff
@@ -159,6 +159,15 @@ const (
 	flagWithNameValues        byte = 0x40
 	flagWithKeyspace          byte = 0x80
 
+	flagValuesV5                uint32 = 0x01
+	flagSkipMetaDataV5                 = 0x02
+	flagPageSizeV5                     = 0x04
+	flagWithPagingStateV5              = 0x08
+	flagWithSerialConsistencyV5        = 0x10
+	flagDefaultTimestampV5             = 0x20
+	flagWithNameValuesV5               = 0x40
+	flagWithKeyspaceV5                 = 0x80
```

Comment:
What do you think about changing only the type of existing flags? 
It allows us to avoid such code duplications. We may implement the new way of constructing the flag field as **uint32**. Also, if the proto version is less than 5, we can just cast it to **byte** inside the corresponding methods like `framer.writeQueryParams`.